samples/dnn/models.yml

```diff
@@ -506,3 +506,20 @@ NAFNet:
   scale: 0.00392
   rgb: true
   sample: "deblurring"
+
+################################################################################
+# Super-resolution models.
+################################################################################
+
+seemoredetails:
+  load_info:
+    url: "https://github.com/Naresh-19/opencv-superres-models/raw/main/seemore_x4v2_static512.onnx"
+    sha1: "584467bd36f5715aa12c3203bba16e4de6392034"
```

Comment:
Could you add link to the original repo with the model, rather than own fork? If it's not possible, please extend your repo with:
- Link to the original model and/or paper.
- Command line / pre-requsites to export ONNX
- License information and credits.